config/packages/dev/storage.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+    Aws\S3\S3Client:
+        arguments:
+            -   version: latest
+                region: '%env(S3_REGION)%'
+                endpoint: 'http://storage:9000'
+                use_path_style_endpoint: true
+                credentials:
```

Comment:
Most of these parameters already exist in the `config/packages/storage.yaml` file, why duplicate them?

Reply:
I only need to add "entrypoint" config in dev mode but in order to add it i have to re-write the whole configuration otherwise this setting would override everything